runtime-builder/tests/integration/test.sh

```diff
@@ -16,7 +16,7 @@
 
 # test.sh deploys the test app and run the integration test on it.
 
-usage() { echo "Usage: $0 <project_id> [builder_tag]"; exit 1; }
+usage() { echo "Usage: $0 <project_id> <builder_image_url>"; exit 1; }
```

Comment:
Change to optional? This can still be default to "gcr.io/${PROJECT_ID}/go1-builder:staging".

Reply:
Done.